pkg/apis/core/types_shoot.go

```diff
@@ -500,6 +500,9 @@ type ClusterAutoscaler struct {
 	// Expander defines the algorithm to use during scale up (default: least-waste).
 	// See: https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/FAQ.md#what-are-expanders.
 	Expander *ExpanderMode
+	// ExpanderConfig defines the configuration of the CA expander
+	// +optional
+	ExpanderConfig *ExpanderConfig `json:"expanderConfig,omitempty"`
```

Comment:
Remove this from internal APIs.
```suggestion
	ExpanderConfig *ExpanderConfig
```